docs/articles/documentation/using-testcafe/programming-interface/index.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: docs
+title: TestCafe Programming Interface
+permalink: /documentation/using-testcafe/programming-interface/
+---
+# TestCafe Programming Interface
+
+This section provides reference for the API that allows you use TestCafe from your NodeJS applications.
+
+## Summary
+
+This part outlines the workflow of using TestCafe programmatically.
```

Comment:
This repeats the sentence above in the shorter form. I'd rather keep the one that above